core/src/main/java/io/aiven/klaw/service/TopicControllerService.java

```diff
@@ -1117,15 +1117,13 @@ private List<TopicInfo> getTopicsPaginated(
     HandleDbRequests handleDbRequests = manageDatabase.getHandleDbRequests();
 
     // To get Producer or Consumer topics, first get all topics based on acls and then filter
-    List<Topic> producerConsumerTopics = new ArrayList<>();
-    if ((AclType.PRODUCER.value.equals(topicType) || AclType.CONSUMER.value.equals(topicType))
-        && teamId != 0) {
-      producerConsumerTopics =
-          handleDbRequests.getAllTopicsByTopictypeAndTeamname(topicType, teamId, tenantId);
+    List<Topic> producerConsumerTopics = null;
+    if ((AclType.PRODUCER.value.equals(topicType) || AclType.CONSUMER.value.equals(topicType))) {
```

Comment:
It is not related but is it possible to change the topicType to enum here https://github.com/Aiven-Open/klaw/blob/main/core/src/main/java/io/aiven/klaw/controller/TopicController.java#L236

Reply:
This one we have an open issue on and we're going to add in a separate PR so the changes are more manageable for integrating with the UI #2216